gradle/libs.versions.toml

```diff
@@ -17,24 +17,24 @@
 # under the License.
 #
 [versions]
-assertj = "3.26.0"
+assertj = "3.26.3"
 # @pin
 caffeine = "2.9.3"
 checkstyle = "8.45.1"
 # @pin
 jackson = "2.14.2"
 # @pin
 jctools = "3.3.0"
-junit-jupiter = "5.11.0-M2"
+junit-jupiter = "5.11.0-RC1"
 licenser = "0.6.1"
 log4j = "2.23.1"
 mockito = "5.12.0"
 pulsar = "3.2.3"
 rat-gradle = "0.8.0"
-reactor = "3.6.7"
+reactor = "3.6.8"
 slf4j = "2.0.13"
-spring-javaformat = "0.0.41"
-testcontainers = "1.19.8"
+spring-javaformat = "0.0.42"
```

Comment:
This is [breaking the build](https://github.com/apache/pulsar-client-reactive/actions/runs/10211256024/job/28256630257#step:6:115) as `0.0.42` [raises the baseline to Java 17](https://github.com/spring-io/spring-javaformat/releases/tag/v0.0.42) (strange version to do this in but I guess that it is the nature of the < `1.0.0` version). 

This also brings up a good question of when should/can we update the baseline for `pulsar-client-reactive` to `17`?

Reply:
thanks, I reverted back to 0.0.41. Let's change the build JVM to Java 17 for 0.6.0 as we have discussed.